app/code/Magento/Ui/Test/Mftf/ActionGroup/AssertAdminErrorMessageDisappearDisabledFieldActionGroup.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminErrorMessageDisappearDisabledFieldActionGroup">
```

Comment:
The ActionGroup does too many things and I don't think it's a reusable one. I would split it in at least into the following ones:
- AdminTogglePriceSwitcherActionGroup
- AdminFocusAndFillElementActionGroup
- AdminAssertErrorIsVisibleActionGroup / AdminAssertErrorIsNotVisibleActionGroup

What do you think about a such refactoring?
Thank you.

Reply:
Thanks. Good idea.